pkgs/build-support/node/fetch-pnpm-deps/default.nix

```diff
@@ -0,0 +1,101 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, nodePackages
+, jq
+, moreutils
+, cacert
+, makeSetupHook
+}: let
+  # The names of these attributes refer to the pnpm lockfile version
+  pinnedVersions = {
```

Comment:
Maybe this attrset could better reflect what it is about.. How about `lockfileVerPnpm`? or `lockVerPnpm`?

Reply:
i would say this name is fitting. after all, the contents of the attribute set *are* the pinned versions. the only issue here is the naming of the attributes, but that's explained in the above comment and i think would be made apparent when used in conjunction with functions that have the versions in their name